src/lib/html.ts

```diff
@@ -152,6 +177,7 @@ function getTreeNodesMap(fileDataMap: FileDataMap): TreeNodesMap {
 
 interface WebTreeMapNode {
   name: string;
+  _name: string;
```

Comment:
Adds new property `_name` to keep the original name of node, as `name` will become something like `name.js • 2.6MB • 33%`.

Reply:
Give it a better name. For example `originalName` or `rawName`